src/dgc_cert.rs

```diff
@@ -6,7 +6,9 @@ pub struct DgcCertName {
     #[serde(default)]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub gn: Option<String>,
-    pub r#fn: String,
+    #[serde(default)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub r#fn: Option<String>,
```

Comment:
Because we are already changing this, I would like to propose this:

```suggestion
    #[serde(skip_serializing_if = "Option::is_none", rename = "fn")]
    pub family_name: Option<String>,
```

This approach could be probably interesting for most of the (cryptic) fields related to the spec, but this is particularly painful because it requires a raw identifier. Feel free to _postpone_ something like this for the future :wink: